src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__product-name {
+  display: flex;
+  max-width: 166px;
+  margin: 40px 0 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card__product-id {
+  display: flex;
+  align-items: flex-start;
+  height: 14px;
+  width: 92px;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  font-size: 10px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__rating {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__rating_stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__rating_stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.card__rating_stars > .card__rating_stars_star:nth-child(-n+4) {
+  background: url("images/star-active.svg") no-repeat center;
+}
```

Comment:
You should add other modifiers for the "stars" block in case to use them later if it would be necessary